CHANGELOG

```diff
@@ -1,3 +1,7 @@
+FUTURE
+
+  - FIX: Add redis version 5 support
```

Comment:
```suggestion
  - FIX: Add redis rubygem version 5 support
```

We need to be clearer here and with the commit message because `Redis 5` may sound like we're referring to Redis instead of the `redis` rubygem.